docs/user/aws/README.md

```diff
@@ -0,0 +1,22 @@
+# AWS Account Set Up
+
+This document is a guide for preparing a new AWS account for use with OpenShift. It 
```

Comment:
There's a bunch of trailing whitespace:

```console
$ git show --oneline --check origin/pr/1030
55ebc12 Adding user doc/guide for AWS account and installation
docs/user/aws/README.md:3: trailing whitespace.
+This document is a guide for preparing a new AWS account for use with OpenShift. It 
docs/user/aws/README.md:4: trailing whitespace.
+will help prepare an account to create a single cluster and provide insight for adjustments which may be 
docs/user/aws/cli.md:3: trailing whitespace.
+The AWS CLI is used to configure your AWS credentials on your system for use by the installer. It can also be useful to 
docs/user/aws/iam.md:3: trailing whitespace.
+In a new AWS account, you are provided with a root user. This account is based on the email address which created 
docs/user/aws/iam.md:4: trailing whitespace.
...
```

You should be able to remove it with:

```console
$ sed -i 's/ *$//' docs/user/aws/*.md
```

Reply:
Sniped and pushed. Thanks!